### DISCUSSION TITLE
Remove text

### DISCUSSION THREAD

**Discussion by space-engineer** [dlemstra/Magick.NET#740] [Help]
hi,
I have two images.
First image
![1](https://user-images.githubusercontent.com/69794397/94051703-95720280-fde0-11ea-9874-22dca2192790.png)

Second image
![2](https://user-images.githubusercontent.com/69794397/94051719-9b67e380-fde0-11ea-8046-83ddf02e14ff.png)

How can I get a result like this?
![3](https://user-images.githubusercontent.com/69794397/94051729-a0c52e00-fde0-11ea-9bc9-890455b8641b.png)

I would be grateful for any help.

**Comment by space-engineer** [comment]
Guys, is this not possible using Magick.NET?

**Reply by dlemstra** [reply]
Nope it's not possible.
